documentation/modules/virt-migration-workflow.adoc

```diff
@@ -30,10 +32,53 @@ The workflow describes the following steps:
 +
 *After the VM disks are transferred:*
 
-. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it.
+. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it when importing from VMWare.
 +
 The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the target VM.
++
+. The `Migration Controller` service creates a `VirtualMachine` CR for each source virtual machine (VM), connected to the PVCs.
 
-. When the target VM is powered on, the `KubeVirt Controller` service creates a `virt-launcher` pod and a `VirtualMachineInstance` CR.
+. If the VM was running on the source environment, the `Migration Controller` powers on the VM, the `KubeVirt Controller` service creates a `virt-launcher` pod and a `VirtualMachineInstance` CR.
 +
 The `virt-launcher` pod runs `QEMU-KVM` with the PVCs attached as VM disks.
+
+*Cold migration from {rhv-short} or {osp} to the local {ocp-short} cluster:*
+
+. When you create a `Migration` custom resource (CR) to run a migration plan, the `Migration Controller` service creates for each source VM disk a `PersistentVolumeClaim` CR, and a `OvirtVolumePopulator` when the source is {rhv-short}, or a `OpenstackVolumePopulator` CR  when the source is {osp}.
++
+*For each VM disk:*
+
+. The `Populator Controller` service creates a temporarily persistent volume claim (PVC).
+. If the `StorageClass` has a dynamic provisioner, the persistent volume (PV) is dynamically provisioned by the `StorageClass` provisioner.
+- The `Migration Controller` service creates a dummy pod to bind *all PVCs*, the name of the pod contains `pvcinit`.
```

Comment:
Consider: Two sentences.  "The ... creates a single dummy pod that bind *all* PVCs. The name of the pod includes `pvcinit`.  If "all PVCs" is a UI name, bold it.